apps/expo/src/app/post/new.tsx

```diff
@@ -0,0 +1,64 @@
+import { router } from 'expo-router';
+import { useState } from 'react';
+import { KeyboardAvoidingView, Platform, Text, TextInput } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context'
+import Button from '~/components/ui/button/button';
+import { useGlobalContext } from '~/context/global-context';
+import { api } from '~/utils/trpc/api';
+
+const inputStyles = {
+  borderWidth: 1,
+  borderColor: '#FFFFFF',
+  borderRadius: 20,
```

Comment:
Border radius should be the same for the input fields as it is for the `Create` button at the bottom.

![Screenshot 2024-04-21 at 1 48 15 PM](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/85365584/0c7c9ec6-5a67-450d-a07d-c62374b941f7)

Reply:
I just took the input styles from another component. Was trying to keep it consistent across screens. But yes I agree, that would look much better.